pkgs/applications/misc/cipher/default.nix

```diff
@@ -42,10 +41,10 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
-  	substituteInPlace data/com.github.arshubham.cipher.desktop.in \
-  		--replace xdg-open ${xdg_utils}/bin/xdg-open
-    chmod +x post_install.py
-    patchShebangs post_install.py
+    substituteInPlace data/com.github.arshubham.cipher.desktop.in \
+                    --replace "gio" "${glib.bin}/bin/gio"
```

Comment:
Think the indentation here is a bit deep, can we use two spaces? Otherwise LGTM.

Reply:
Guilty, I copy/paste you're suggestion :sweat_smile: